OrbitCore/TracepointEventBuffer.h

```diff
@@ -37,8 +37,12 @@ class TracepointEventBuffer {
     }
   }
 
+  uint32_t GetNumTracepointsForThreadId(int32_t thread_id) const;
+
  private:
   int32_t kNotTargetProcessThreadId = -2;
+  int32_t no_total_tracepoints_ = 0;
```

Comment:
I would use `num_total_tracepoints_` or `total_tracepoint_count_`, `no` sounds more like&mdash;well&mdash;"no" than like "number".

Reply:
I see what you mean. Thanks a lot! I will update the code.